src/desktop/apple_notification.mm

```diff
@@ -44,6 +45,7 @@ void send_notification(const std::string& owner, const std::string& message, con
     }
 }
 
+#pragma clang diagnostic pop
```

Comment:
Nitpicking: this should be one line higher (attached to `send_notification()`, not `send_cocoa_notification()`).